packages/main/src/StepInput.js

```diff
@@ -0,0 +1,651 @@
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-step-input</code> is displayed as read-only.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		readonly: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines a short hint, intended to aid the user with data entry when the
+		 * <code>ui5-step-input</code> has no value.
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When no placeholder is set, the format pattern is displayed as a placeholder.
+		 * Passing an empty string as the value of this property will make the <code>ui5-step-input</code> appear empty - without placeholder or format pattern.
+		 *
+		 * @type {string}
+		 * @defaultvalue undefined
+		 * @public
+		 */
+		placeholder: {
+			type: String,
+			defaultValue: undefined,
+		},
+
+		/**
+		 * Determines the name with which the <code>ui5-step-input</code> will be submitted in an HTML form.
+		 *
+		 * <br><br>
+		 * <b>Important:</b> For the <code>name</code> property to have effect, you must add the following import to your project:
+		 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When set, a native <code>input</code> HTML element
+		 * will be created inside the <code>ui5-step-input</code> so that it can be submitted as
+		 * part of an HTML form. Do not use this property unless you need to submit a form.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		name: {
+			type: String,
+		},
+
+		/**
+		 * Determines the number of digits after the decimal point of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		valuePrecision: {
+			type: Integer,
+			defaultValue: 0,
+		},
+
+		/**
+		 * Defines the aria-label attribute for the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
+			type: String,
+		},
+
+		/**
+		 * Receives id(or many ids) of the elements that label the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		ariaLabelledby: {
+			type: String,
+			defaultValue: "",
+		},
+
+		_decIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_incIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_focused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_inputFocused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_previousValue: {
+			type: Float,
+			noAttribute: true,
+		},
+
+		_previousValueState: {
+			type: String,
+			noAttribute: true,
+			defaultValue: "",
+		},
+
+		_waitTimeout: {
+			type: Float,
+			noAttribute: true,
+		},
+
+		_speed: {
+			type: Float,
+			noAttribute: true,
+		},
+
+		_btnDown: {
+			type: Boolean,
+			noAttribute: true,
+			defaultValue: false,
```

Comment:
Delete this line to remove the console warning

Reply:
Done